File: PROGRAMMERS/PY/lv.1/최댓값과최솟값.py
# s = "1 2 3 4"
s = "-1 -2 -3 -4"
# s = "-1 -1"

def solution(s):
    answer = ''
    s_list = list(map(int, s.split(' ')))
    s_list.sort()
    answer += str(s_list[0])
    answer += " "
    answer += str(s_list[-1])
    return answer

print(solution(s))

File: PROGRAMMERS/JAVA/src/Lv_2/최댓값과최솟값.java
package Lv_2;

import java.util.Arrays;

public class 최댓값과최솟값 {
    public static String solution(String s) {
        String answer = "";
        int[] arr = Arrays.asList(s.split(" ")).stream().mapToInt(Integer::parseInt).toArray();
        Arrays.sort(arr);
        answer += Integer.toString(arr[0]) + " " + Integer.toString(arr[arr.length - 1]);
        return answer;
    }

    public static void main(String[] args) {
        String result = solution("1 2 3 4");
    }
}
